CONTRIBUTING.md

```diff
@@ -66,6 +67,8 @@ _For larger features, your contribution will have a higher likelihood of getting
 
 ### Post-Merge checklist
 - [ ] You have tagged the release using `git tag v<version_number>`.
+- [ ] You have pushed this tag using `git push <tag_name>`
```

Comment:
This push to origin, only contributors can push to origin. So it should state that the merger have to tag.